tsconfig.json

```diff
@@ -1,18 +1,20 @@
 {
-    "compilerOptions": {
-      "target": "esnext",
-      "module": "esnext",
-      "moduleResolution": "node",
-      "skipLibCheck": true,
-  
-      "allowJs": true,
-      "checkJs": true,
-      "strict": true,
-  
-      "noEmit": true,
-    },
-    "include": [
-      ".eslintrc.cjs",
-    ]
-  }
-  
\ No newline at end of file
+  "compilerOptions": {
+    "target": "esnext",
+    "module": "esnext",
+    "moduleResolution": "node",
+    "skipLibCheck": true,
+
+    "allowJs": true,
+    "checkJs": true,
```

Comment:
happy to see this is the default so it highlights what packages still need to be made compatible with it